_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: Image accessible name is filename
+description: |
+  This rule checks that image elements that use their source filename as their accessible name do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input type="image"` or `area` elements, or any HTML element with the semantic role of `img`, where the element is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an [accessible name](#accessible-name) that matches the filename specified in the `src` attribute.
```

Comment:
`input type="image"` is somewhat ambiguous. Should the rule only apply to `input` elements with a `type` attribute that is exactly equal to `"image"`? Or should it really be applicable to all `input` elements that are image buttons, i.e. has a `type` attribute whose value maps to the `Image Button` state?